catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -111,93 +106,49 @@ public Metacard transform(InputStream input) throws IOException, CatalogTransfor
     @Override
     public Metacard transform(InputStream input, String id)
             throws IOException, CatalogTransformerException {
-        if (contentMetadataExtractors.isEmpty()) {
-            return transformWithoutExtractors(input, id);
-        } else {
-            return transformWithExtractors(input, id);
-        }
+        return transformWithoutExtractors(input, id);
+    }
+
+    /**
+     * @param function must be non-null
+     */
+    void setInputStreamPDDocumentFunction(CheckedFunction<InputStream, PDDocument> function) {
+        notNull(function, "function must be non-null");
```

Comment:
If we keep those only for unit testing, do we need the comments and `null` checks?